integration-libs/opf/components/opf-checkout-payments/opf-checkout-payments.component.ts

```diff
@@ -26,6 +26,9 @@ export class OpfCheckoutPaymentsComponent {
       })
     );
 
+  @Input()
+  disabled = false;
```

Comment:
If I understand it correctly, the user cannot select a payment option before accepting terms and conditions - in this case `disabled` should have a `true` value by default.

Reply:
Right, thanks!